x/interchainstaking/types/rebalance.go

```diff
@@ -1,6 +1,7 @@
 package types
 
 import (
+	fmt "fmt"
```

Comment:
idk why this happened
```suggestion
     "fmt"
```

Reply:
lol